cpu/stm32f1/periph/spi.c

```diff
@@ -275,20 +288,23 @@ void spi_poweroff(spi_t dev)
     switch (dev) {
 #if SPI_0_EN
         case SPI_0:
+            while ((SPI_0_DEV->SR & SPI_SR_BSY)) {}
```

Comment:
I don't see it critical as well. I've added it for safety reasons. In the [referance manual](http://www.st.com/content/ccc/resource/technical/document/reference_manual/59/b9/ba/7f/11/af/43/d5/CD00171190.pdf/files/CD00171190.pdf/jcr:content/translations/en.CD00171190.pdf) page 721 sec. *25.3.8 Disabling the SPI* it is said "Then wait until BSY=0"